{"sections":[],"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError"]]},"schemaVersion":{"patch":0,"major":0,"minor":3},"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/noCredentials"},"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"noCredentials"},{"kind":"text","text":": "},{"preciseIdentifier":"s:5Auth023CredentialsManagerErrorV","kind":"typeIdentifier","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError","text":"CredentialsManagerError"}],"platforms":["iOS"]}],"kind":"declarations"}],"metadata":{"title":"noCredentials","modules":[{"name":"Auth0"}],"roleHeading":"Type Property","externalID":"s:5Auth023CredentialsManagerErrorV02noB0ACvpZ","symbolKind":"property","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"noCredentials"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:5Auth023CredentialsManagerErrorV","text":"CredentialsManagerError"}],"role":"symbol"},"variants":[{"paths":["\/documentation\/auth0\/credentialsmanagererror\/nocredentials"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"text":"No credentials were found in the store.","type":"text"},{"text":" ","type":"text"},{"text":"This error does not include a ","type":"text"},{"type":"reference","isActive":true,"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi"},{"type":"text","text":"."}],"references":{"doc://com.auth0.Auth0/documentation/Auth0":{"role":"collection","abstract":[{"text":"SDK for Apple platforms.","type":"text"}],"type":"topic","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","kind":"symbol","url":"\/documentation\/auth0","title":"Auth0"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError/noCredentials":{"abstract":[{"type":"text","text":"No credentials were found in the store."},{"text":" ","type":"text"},{"text":"This error does not include a ","type":"text"},{"isActive":true,"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","type":"reference"},{"text":".","type":"text"}],"url":"\/documentation\/auth0\/credentialsmanagererror\/nocredentials","title":"noCredentials","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"noCredentials","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:5Auth023CredentialsManagerErrorV","text":"CredentialsManagerError","kind":"typeIdentifier"}],"type":"topic","role":"symbol","kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError\/noCredentials"},"doc://com.auth0.Auth0/documentation/Auth0/CredentialsManagerError":{"title":"CredentialsManagerError","navigatorTitle":[{"text":"CredentialsManagerError","kind":"identifier"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/CredentialsManagerError","role":"symbol","url":"\/documentation\/auth0\/credentialsmanagererror","type":"topic","abstract":[{"text":"Represents an error during a Credentials Manager operation.","type":"text"}],"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"CredentialsManagerError","kind":"identifier"}],"kind":"symbol"},"doc://com.auth0.Auth0/documentation/Auth0/Auth0Error/cause-9wuyi":{"type":"topic","abstract":[{"type":"text","text":"The underlying "},{"type":"codeVoice","code":"Error"},{"type":"text","text":" value, if any. Defaults to "},{"type":"codeVoice","code":"nil"},{"type":"text","text":"."}],"kind":"symbol","url":"\/documentation\/auth0\/auth0error\/cause-9wuyi","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"cause","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:s5ErrorP","text":"Error","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"title":"cause","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/Auth0Error\/cause-9wuyi","role":"symbol"}}}